understanding/identify-purpose.html

```diff
@@ -55,143 +53,115 @@
-                  of icons, regions and components (such as buttons, links, and fields) so that user
-                  agents can determine the purpose of each and adapt indicators or terminology to make
-                  them understandable for the user. It is achieved by adding semantics or metadata that
-                  provide this context. It is similar to adding role information (as required by 4.1.2)
-                  but instead of providing information about what the UI component is (such as an image)
-                  it provides information about what the component represents (such as a link to the
-                  home page).
+               <p>この達成基準は、コンテンツ制作者がアイコン、リージョン、コンポーネント (ボタン、リンク、フィールドなど) の目的をプログラムにより関連付けて、ユーザエージェントがそれぞれの目的を判断し、インジケーター又は用語を利用者に理解できるように適応させることを要求する。それは、このコンテキストを提供するセマンティクス又はメタデータを追加することで達成される。役割 (role) の情報を追加するのと似ているが (4.1.2 で要求されているように) 、ユーザインタフェース コンポーネントが何であるかについての情報 (例えば画像) を提供する代わりに、そのコンポーネントが何を表すものかについての情報 (例えばホームページへのリンクなど) を提供する。
                </p>
-               <p>Identifying regions of the page allows people to remove or highlight regions with
-                  their user agent.
+               <p>ページの領域を特定することで、利用者は自分のユーザエージェントで領域を削除又は強調表示することができる。
                </p>
-               <p>Products for people who are non-vocal often use symbols to help users communicate.
-                  These symbols are in fact people's language. Unfortunately, many of these symbols
-                  are both subject to copyright and not interoperable. That means end users can only
-                  use one device, and cannot use content, apps, or assistive technologies that have
-                  not been made by a single company.
+               <p>声を出さない人向けの製品では、コミュニケーションを促進するために、よくシンボルを使用する。これらのシンボルは事実上、人々の伝達手段としての言語である。残念ながら、これらのシンボルの多くは著作権の対象であるだけでなく、相互運用もできない。つまり、エンドユーザは一つのデバイスしか使用できず、 1 社だけで作られていないコンテンツ、アプリケーション、又は支援技術を使用できない。
                </p>
-               <p>This Success Criterion enables symbols to be interoperable so that symbol users can
-                  understand different content that was not just made by one company.  When users' symbols
-                  are mapped to the same nodes, then user agents can load the user-understandable symbol.
-                  People can then buy the symbols and use them across different devices or applications.
-                  (Note that the symbols would still be proprietary, but they could then be interoperable.)
+               <p>この達成基準により、シンボルは相互運用可能になり、シンボルの利用者は 1 社だけで作られていない様々なコンテンツを理解できる。利用者のシンボルが同じノードに割り当てられている場合、ユーザエージェントは利用者が理解可能なシンボルを取得できる。人々はシンボルを購入し、それらを様々なデバイスやアプリケーションで使用することができる。 (シンボルはまだ独自のものであることに注意したいが、相互運用可能なものもある。)
                </p>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
-               <p>People who benefit have many different cognitive disabilities including:</p>
+               <h2>利点</h2>
+               <p>恩恵を受ける人々は次のような認知障害を持っている:</p>
```

Comment:
「障害を持つ」になってしまっていることに気づいたので、しれっと直しておきましょうか
```suggestion
               <p>恩恵を受ける人々は次のようなさまざまな認知障害がある:</p>
```